README.md

```diff
@@ -13,19 +13,19 @@ dependencies: ["https://github.com/DataDog/serverless-plugin-datadog/blob/master
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue)](https://github.com/DataDog/serverless-plugin-datadog/blob/master/LICENSE)
 
 Datadog recommends the Serverless Framework Plugin for developers using the Serverless Framework to deploy their serverless applications.
-The plugin automatically enables instrumentation for your Python and Node.js applications to collect metrics, traces, and logs by:
+The plugin automatically enables instrumentation for applications to collect metrics, traces, and logs by:
 
 - Installing the Datadog Lambda library to your Lambda functions as a Lambda layer.
 - Installing the Datadog Lambda Extension to your Lambda functions as a Lambda layer (`addExtension`) or subscribing the Datadog Forwarder to your Lambda functions' log groups (`forwarderArn`).
-- Making the required configuration changes, such as adding environment variables, to your Lambda functions.
+- Making the required configuration changes, such as adding environment variables or additional tracing layers, to your Lambda functions.
 
 ## Getting started
 
-To quickly get started, follow the installation instructions for [Python][1] or [Node.js][2], and view your function's enhanced metrics, traces, and logs in Datadog. These instructions will get you a basic working setup.
+To quickly get started, follow the installation instructions for [Python][1], [Node.js][2], [Ruby][3], [Java][4], [Go][5], or [.NET][6] and view your function's enhanced metrics, traces, and logs in Datadog. These instructions will get you a basic working setup. If working in Ruby, Java, or Go, further code changes are required.
```

Comment:
Hi @lizapressman, thanks for working on the integration for other environments than Pyhton and Node only. This is exactly what we're were looking for, in our case we use Java/Kotlin and would be happy to use this serverless plugin directly. What exactly code changes are required for us, can you maybe provide some more details on that?

Reply:
I'm working on updating our public docs right now so there will be more information there as soon as this is released. As of now, for Java, this really only adds the extension so much of the changes will be the same as they are now. But, for specific details, I recommend checking our [public docs](https://docs.datadoghq.com/serverless/installation/java?tab=custom) (probably/hopefully) early next week

Reply:
Hi @lizapressman, thanks for the further infos. I'll check the documentation next week. In the meantime, based on your dotnet integration and the current documentation, I've also added the needed environment variable and trace layer for Java in #244, would be nice if you could take a look on it. Thanks.

Reply:
@drobakowski Thanks for adding that, I will take a look!